### PR TITLE
Add impact severity

### PR DESCRIPTION
**Pull request by moonso**
- Added impact severity filter for vcf and gemini
- Optimized how to get one variant in vcf adapter(should be alot faster now)
- Changed how vcf adapter creates variants

### CODE REVIEW

**Comment by moonso** [comment]
Please @robinandeer  or @guillermo-carrasco or both?

File: puzzle/utils/constants.py [hunk 1]
```diff
@@ -49,6 +49,63 @@
   'intergenic_variant'
 )
 
+IMPACT_SEVERITIES = {
```
Comment (robinandeer) [inline puzzle/utils/constants.py]: This is the same as `SO_TERMS `, no? If so we could avoid repeating them and just get them as the `IMPACT_SEVERITIES.keys()` given that we make it into an `OrderedDict`

File: puzzle/blueprints/variants/views.py [hunk 2]
```diff
@@ -2,7 +2,8 @@
 from flask import (abort, current_app as app, Blueprint, render_template,
                    request)
 
-from puzzle.constants import INHERITANCE_MODELS_SHORT, SO_TERMS, SV_TYPES
+from puzzle.constants import (INHERITANCE_MODELS_SHORT, SO_TERMS, SV_TYPES,
+                            IMPACT_LEVELS)
```
Comment (robinandeer) [inline puzzle/blueprints/variants/views.py]: OCD but we just need 2 spaces more to align :sweat_smile:

File: puzzle/plugins/gemini/variant_mixin.py [hunk 3]
```diff
@@ -289,6 +307,9 @@ def _format_variant(self, gemini_variant, individual_objs, index=0):
 
         #Add the most severe consequence
         variant['most_severe_consequence'] = gemini_variant['impact_so']
+        
+        #Add the impact severity
+        variant['impact_severity'] = gemini_variant['impact_severity']
```
Comment (robinandeer) [inline puzzle/plugins/gemini/variant_mixin.py]: Nice ^^

**Comment by robinandeer** [comment]
I would need an introduction to this code I realize :)

I'll pull and test that stuff seems to work soon

File: puzzle/blueprints/variants/views.py (hunk 2 above)
Comment (moonso) [inline puzzle/blueprints/variants/views.py]: Haha

File: puzzle/plugins/gemini/variant_mixin.py (hunk 3 above)
Comment (moonso) [inline puzzle/plugins/gemini/variant_mixin.py]: :+1:

**Comment by moonso** [comment]
Ok just tell me and we can sit down together

**Pull request merged**